src/test/kotlin/apply/RecruitmentFixtures.kt

```diff
@@ -2,15 +2,17 @@ package apply
 
 import apply.application.RecruitmentData
 import apply.application.RecruitmentItemData
+import apply.application.TermSelectData
 import apply.domain.recruitment.Recruitment
 import apply.domain.recruitmentitem.RecruitmentItem
 import apply.ui.admin.recruitment.RecruitmentForm
 import apply.ui.admin.recruitment.RecruitmentItemForm
 import java.time.LocalDateTime
 
 private const val RECRUITMENT_TITLE: String = "웹 백엔드 3기"
-private val START_DATE_TIME: LocalDateTime = LocalDateTime.MIN
-private val END_DATE_TIME: LocalDateTime = LocalDateTime.MAX
+private val SINGLE_TERM: TermSelectData = TermSelectData("단독 모집")
```

Comment:
```suggestion
private val SINGLE_TERM: TermSelectData = TermSelectData(Term.SINGLE)
```
이왕이면 "단독 모집"의 의미를 가지는 값을 `companion object` 값을 써주는 건 어떨까요?
같은 의미의 문자열보다는 해당 내용이 변경에 더 잘 대응할 수 있을 것 같아요

Reply:
테스트 데이터이므로 꼭 '단독 모집'이 아니어도 됩니다. `SINGLE_TERM`이라는 이름은 오해의 소지가 있어 `TERM`으로 변경했습니다.